GETTING_STARTED.md

```diff
@@ -72,7 +72,7 @@ In your python interpretor:
         }
     }
 >>> from vissl.utils.io import save_file
->>> save_file(json_data, "/tmp/configs/config/dataset_catalog.json")
+>>> save_file(json_data, "/tmp/configs/config/dataset_catalog.json",append_to_json=True)
```

Comment:
nit: add space:  `json_data, "/tmp/configs/config/dataset_catalog.json", append_to_json=True`(and for all the others).

Reply:
These should now be ```append_to_json=False```, since we want to prevent the JSON decoder error.